File: backend/src/main/java/space/davids_digital/sweetie/rest/dto/DashboardDto.java
package space.davids_digital.sweetie.rest.dto;

import java.util.List;

public record DashboardDto(
        UserDto user,
        List<Chat> chats
) {
    public record Chat(
            long peerId,
            String title,
            String pictureUrl,
            boolean botEnabled
    ) {}
}

File: backend/src/main/java/space/davids_digital/sweetie/rest/dto/DashboardDto.kt
package space.davids_digital.sweetie.rest.dto

data class DashboardDto(
    val user: UserDto,
    val chats: List<Chat>
) {
    data class Chat(
        val peerId: Long,
        val title: String,
        val pictureUrl: String,
        val botEnabled: Boolean
    )
}